api/documentation/src/pages/change-logs.md

```diff
@@ -13,6 +13,10 @@ title: Pass Culture API change logs
 **You can find a migration tutorial [here](/docs/tutorials/migrate-to-the-new-api).**
 :::
 
+## December 2024
+
+- The `subcategoryId` field has been removed from collective offers. You must now use the `formats` field to specify the educational format of your collective offer. The `formats` field is now required when creating a collective offer.
```

Comment:
p-e préciser les endpoints concernés comme en dessous : le champ n'apparait plus sur les GET collective offer et GET collective offers + n'est plus accepté pour les routes PATCH et le POST (en mettant un lien comme c'est fait au changelog `You can edit the name of an event ...`)